dpnp/backend/queue_sycl.cpp

```diff
@@ -118,7 +119,39 @@ void backend_sycl::backend_sycl_queue_init(QueueOptions selector)
     std::cout << "SYCL kernels link time: " << time_kernels_link.count() << " (sec.)\n" << std::endl;
 }
 
+void backend_sycl::backend_sycl_rng_engine_init()
+{
+    if (rng_engine)
+    {
+        backend_sycl::destroy_rng_engine();
+    }
+    // TODO:
+    // choose engine as is in numpy
+    // seed number
+    size_t seed = std::time(nullptr);
+    rng_engine = new mkl_rng::philox4x32x10(DPNP_QUEUE, seed);
+}
+
+void backend_sycl::backend_sycl_rng_engine_init(size_t seed)
+{
+    if (rng_engine)
+    {
+        backend_sycl::destroy_rng_engine();
+    }
+    rng_engine = new mkl_rng::philox4x32x10(DPNP_QUEUE, seed);
+}
+
 void dpnp_queue_initialize_c(QueueOptions selector)
 {
     backend_sycl::backend_sycl_queue_init(selector);
 }
+
+void dpnp_srand_c()
```

Comment:
Are you sure we need two functions with and without argument?
I would propose to use one function and use some `std::time(0)` as initializer at python level

Reply:
@shssf Of course, we can add initializing on to python level and use functions with default arguments, but then we will not have default (std::time(0) initializing ) for the backend interface. If it is needed, I can change it.